projects/view/client/singleProject/singleProjectDetailed.css

```diff
@@ -0,0 +1,645 @@
+/*  SECTIONS  */
```

Comment:
In this, and all, file(s), use two spaces per level of indentation. Convert tabs to spaces.